VolleyPaint/Assets/Scripts/Ability/SpikeBall.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(menuName ="Litkey/Ability/SpikeBall")]
+public class SpikeBall : Ability
+{
+    Transform ball;
+
+    // stops the ball for a second and spikes the ball to the ground diagnally
+    public override void UseAbility(GameObject parent)
+    {
+        base.UseAbility(parent);
+        ball = GameObject.FindGameObjectWithTag("Ball").transform;
```

Comment:
have you finished implementing this or is this something you scrapped/stopped working on?

Reply:
this ball ability should probably behave differently. I think we need to talk about this. 
So I'd say work on later.